lib/ui/compositing/scene.cc

```diff
@@ -36,8 +38,13 @@ fml::RefPtr<Scene> Scene::create(std::shared_ptr<flutter::Layer> rootLayer,
 Scene::Scene(std::shared_ptr<flutter::Layer> rootLayer,
              uint32_t rasterizerTracingThreshold,
              bool checkerboardRasterCacheImages,
-             bool checkerboardOffscreenLayers)
-    : m_layerTree(new flutter::LayerTree()) {
+             bool checkerboardOffscreenLayers) {
+  auto viewport_metrics = UIDartState::Current()->window()->viewport_metrics();
+
+  m_layerTree = std::make_unique<LayerTree>(
```

Comment:
As we're doing the clean up, maybe it's a good time to conform to the style by changing `m_layerTree` to `layer_tree_`. @chinmaygarde : is there any particular reason that we have a different naming style here?

Reply:
Done, unless chinmay mentions a good reason